src/style.css

```diff
@@ -0,0 +1,74 @@
+@font-face {
+  font-family: "Avenir", sans-serif;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  box-sizing: border-box;
+  font-family: "Avenir", sans-serif;
+  font-weight: lighter;
+}
+
+.search {
+  margin: 20px 0;
+  position: relative;
+}
+
+.queryy {
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  font-style: normal;
+  font-weight: normal;
+  color: #3d4e61;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.search-icon {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+  height: 19px;
+  width: 19px;
+}
+
+.query-second {
+  font-family: "Avenir", sans-serif;
+  width: 100%;
+  height: 42px;
+  padding-left: 33px;
+  font-style: normal;
+  font-weight: normal;
+  color: #3d4e61;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  font-size: 14px;
+  line-height: 20px;
+}
```

Comment:
do not duplicate styles that already in `.queryy` class
same for other selectors